prog/dftb+/lib_common/globalenv.F90

```diff
@@ -0,0 +1,113 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2017  DFTB+ developers group                                                      !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+#:include 'common.fypp'
+
+!> Contains fundamental global environment settings
```

Comment:
Should clarify in what sense this is  global, as these are computing environment parameters.

Reply:
Done. I am also happy to rename it, if you have any suggestions.